shop/src/main/java/ro/msg/learning/shop/repo/OrderRepo.java

```diff
@@ -0,0 +1,12 @@
+package ro.msg.learning.shop.repo;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Component;
+import ro.msg.learning.shop.entity.Order;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Component
```

Comment:
have a look at this: https://www.baeldung.com/spring-component-repository-service
Now, component is not technically wrong, but we use it to define custom components. Using @Repository is like using @Component, but Spring will know exactly what exceptions to throw and how to behave with this specific component